### DISCUSSION TITLE
希望能尽早推出运行环境PHP-FPM FOR DEBIAN基础镜像

### DISCUSSION THREAD

**Discussion by zhaozhaohong** [1Panel-dev/1Panel#5200] [Ideas]
### 1Panel 版本

1.9

### 请描述您的需求或者改进建议

PHP运行环境默认是APLINE基础镜像，该系统非GLIB库有的知名程序特别是电商类的会有页面提示错误，加载不出下拉框设置。

**Comment by zhengkunwang223** [comment]
可以先看一下这个解决方案 https://bbs.fit2cloud.com/t/topic/5201/6
